code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1365,6 +1366,17 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 
 	apply_overlay(HALO_LAYER)
 
+/mob/living/carbon/human/proc/update_eyes_overlay_layer()
+	remove_overlay(EYES_OVERLAY_LAYER)
+
+	var/obj/item/organ/internal/eyes/eyes_organ = src.get_int_organ(/obj/item/organ/internal/eyes)
+	if(eyes_organ && istype(eyes_organ, /obj/item/organ/internal/eyes/cybernetic/eyesofgod))
```

Comment:
```suggestion
	if(]istype(eyes_organ, /obj/item/organ/internal/eyes/cybernetic/eyesofgod))
```
`istype()` will null check the target datum for you and return FALSE if its null